### ISSUE TITLE
Backpack/PRO is installing/cloning from Github instead of intended backpack repository

### ISSUE THREAD

**Issue by thijsrijkers** [Laravel-Backpack/community-forum#513]
Im trying to install backpack but it keeps asking for github authentication because its targeting the github repo instead of the backpack one.

My composer.json:

```
{
    "name": "laravel/laravel",
    "type": "project",
    "description": "The Laravel Framework.",
    "keywords": ["framework", "laravel"],
    "license": "MIT",
    "require": {
        "php": "^8.1",
        "backpack/crud": "^5.0.0",
        "backpack/permissionmanager": "^6.0",
        "backpack/pro": "1.6.3",
        "barryvdh/laravel-dompdf": "^2.0",
        "beyondcode/laravel-websockets": "^1.14",
        "darkaonline/l5-swagger": "^8.0",
        "devianl2/laravel-scorm": "*",
        "fruitcake/laravel-cors": "^2.0",
        "guzzlehttp/guzzle": "^7.0.1",
        "laravel/framework": "^9.0",
        "laravel/sanctum": "^2.12",
        "laravel/socialite": "^5.5",
        "laravel/telescope": "^4.6",
        "laravel/tinker": "^2.5",
        "league/flysystem-aws-s3-v3": "^3.0",
        "pusher/pusher-php-server": "^7.2",
        "reliese/laravel": "^1.1",
        "socialiteproviders/microsoft": "^4.1",
        "spatie/laravel-medialibrary": "^10.7",
        "spatie/laravel-permission": "^5.4"
    },
    "require-dev": {
        "backpack/generators": "^3.1",
        "spatie/laravel-ignition": "^1.0",
        "fakerphp/faker": "^1.9.1",
        "laravel/sail": "^1.0.1",
        "mockery/mockery": "^1.4.4",
        "nunomaduro/collision": "^6.1",
        "phpunit/phpunit": "^9.5.10"
    },
    "autoload": {
        "psr-4": {
            "App\\": "app/",
            "Database\\Factories\\": "database/factories/",
            "Database\\Seeders\\": "database/seeders/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "scripts": {
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover --ansi"
        ],
        "post-update-cmd": [
            "@php artisan vendor:publish --tag=laravel-assets --ansi --force"
        ],
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate --ansi"
        ]
    },
    "extra": {
        "laravel": {
            "dont-discover": []
        }
    },
    "config": {
        "optimize-autoloader": true,
        "preferred-install": "dist",
        "sort-packages": true,
        "disable-tls": false
    },
    "minimum-stability": "dev",
    "prefer-stable": true,
    "repositories":
    [
        {
            "type": "composer",
            "url": "https://repo.backpackforlaravel.com/"
        }
    ]
}
```

**Comment by pxpm** [comment]
Hello @thijsrijkers can you please post the error ? 
Thanks

**Comment by tabacitu** [comment]
Hi @thijsrijkers ,

Just checked - our repo is working ok, it's not down or anything.

What I think is happening is that when you are trying to download from repo.backpackforlaravel.com you are getting Error 402 - which means "Payment required". That means that you do not have access to that update. For example your 12 months of updates might have included PRO 1.6.3, but not 1.6.4. 

What you can do:
- (a) Purchase another 12 months of access for that package on our website.
- (b) Lock your composer.json to the last version you have access to (no purchase). You can see the last version you have access to in the email you received when you got that error.

Right now figuring out the last version is a little clunky outside that email. But you can figure it out because:
- your [tokens page](https://backpackforlaravel.com/user/tokens) shows your expiration date
- the [release notes](https://backpackforlaravel.com/products/pro-for-unlimited-projects/CHANGELOG.md) show when each version was released

Let us know if that fixes it for you!
